packages/cli/src/commands/remove/index.ts

```diff
@@ -211,7 +210,7 @@ export default async function remove(client: Client) {
   });
   const start = Date.now();
 
-  await Promise.all<any>([
+  await Promise.all<boolean | ProjectNotFound | undefined>([
```

Comment:
Does this even need to be typed? The return value is not being used

Reply:
no, I removed it.